src/Compilers/Core/Portable/Symbols/TypeCompareKind.cs

```diff
@@ -10,13 +10,16 @@ namespace Microsoft.CodeAnalysis
     [Flags]
     internal enum TypeCompareKind
     {
-        ConsiderEverything = 0,
+        ConsiderEverything = 0, // PROTOTYPE(NullableReferenceTypes): Does this include comparing nullable modifiers? If not, rename.
         IgnoreCustomModifiersAndArraySizesAndLowerBounds = 1,
         IgnoreDynamic = 2,
         IgnoreTupleNames = 4,
         IgnoreDynamicAndTupleNames = IgnoreDynamic | IgnoreTupleNames,
+        CompareNullableModifiersForReferenceTypes = 8,
+        UnknownNullableModifierMatchesAny = 16, // Has no impact without CompareNullableModifiersForReferenceTypes
+        AllAspects = CompareNullableModifiersForReferenceTypes, // PROTOTYPE(NullableReferenceTypes): Remove if not used.
         AllIgnoreOptions = IgnoreCustomModifiersAndArraySizesAndLowerBounds | IgnoreDynamic | IgnoreTupleNames,
-        AllIgnoreOptionsForVB = IgnoreCustomModifiersAndArraySizesAndLowerBounds | IgnoreTupleNames
+        AllIgnoreOptionsForVB = AllIgnoreOptions // PROTOTYPE(NullableReferenceTypes): Can be removed and replaced with AllIgnoreOptions.
```

Comment:
Reverted. But is there anything preventing a caller in VB from using `AllIgnoreOptions` instead of `AllIgnoreOptionsForVB` even with the extra enum value?